src/VirtoCommerce.CatalogModule.Data/Search/Indexing/AggregationConverter.cs

```diff
@@ -235,11 +236,16 @@ protected virtual IList<AggregationItem> GetAttributeAggregationItems(IList<Aggr
         {
             var result = aggregationResponseValues
                 .Select(v =>
-                    new AggregationItem
+                {
+                    return new AggregationItem
                     {
-                        Value = v.Id,
+                        Value = v.Id.Contains('_') ? v.Id.Split('_')[0] : v.Id,
```

Comment:
Having split by '_' here looks a bit dangerous for me, as now any other attribute containing '_' in index value will be changed that way - it could lead to non-working aggregations. Maybe need to limit such parsing appliance somehow.

Reply:
have added outline delimiter with '____'